src/components/pages/Splash/index.tsx

```diff
@@ -40,8 +40,8 @@ export default function Splash() {
                   </Button>
                   <Button
                     shape="round"
-                    size="large"
-                    style={{ width: '16vw', height: '3.75vw', fontWeight: 'bold', fontSize: '1.2vw' }}
+                    /* Added class for button & removed size attr since edits in html supersede all CSS */
```

Comment:
Great refactor here, i hate inline styling even though i do it sometimes to be quick. Theres no need to leave comments in the repo like this tho, put messages like this in the pull request